### PR TITLE
AHTI-412 | Update project information on README

### PR DESCRIPTION
**Pull request by charn**
Update the project README to conform with documentation practices.

Refs AHTI-331, AHTI-412

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Contributions are accepted as pull requests.  Please observe our coding
+practices at https://github.com/City-of-Helsinki/bestpractice/ .
+Please make your pull requests short, elegant and only handling one
+issue at a time!
+
+If you make a pull request, you may also want to contact
+<INSERT SERVICE'S ROLE ADDRESS> to tell about your contribution.
```
Comment (charn) [inline CONTRIBUTING.md:7]: I wonder if there's an email we could use here. :thinking:

Comment (igordavydsson) [reply CONTRIBUTING.md:7]: I think there is a [gitter chat room](https://gitter.im/City-of-Helsinki/heldev), we could mention it here. OR just delete this paragraph 🤷

File: README.md [hunk 2]
```diff
@@ -96,3 +111,34 @@ Basic `black` commands:
...
+
+## Issue tracking
+
+* [Github issue list](https://github.com/City-of-Helsinki/ahti/issues)
+* [Jira issues](https://helsinkisolutionoffice.atlassian.net/projects/AHTI/issues/?filter=allissues)
+
+
+## API documentation
+
+View the API documentation by visiting: http://localhost:8082/graphql and see
```
Comment (igordavydsson) [inline README.md:130]: we could also give a link to https://ahti-api.test.kuva.hel.ninja/graphql here? probably first step for most devs is to look at some running test instance docs if possible

File: CONTRIBUTING.md (hunk 1 above)
Comment (igordavydsson) [reply CONTRIBUTING.md:7]: so, what do you think we should do here?

Comment (charn) [reply CONTRIBUTING.md:7]: I'll remove it for now. I shall add it back when I figure out the proper thing to put there.

**Pull request merged**